src/api/admin/types.ts

```diff
@@ -654,6 +654,27 @@ export type AddAgentInfoRequest = { agent_infos: Array<string> };
  */
 export type AddAgentInfoResponse = unknown;
 
+/**
+ * @public
+ */
+export type AgentMetaInfoRequest = { url: string; dna_hashes?: DnaHash[] };
```

Comment:
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

"use interfaces until you need to use a type"
```suggestion
export interface AgentMetaInfoRequest = { url: string; dna_hashes?: DnaHash[] };
```

Reply:
https://github.com/holochain/holochain-client-js/pull/353/commits/50ca7f8bba6d834d514fc90a23d99d4b873fa2b6